x-pack/plugins/apm/public/components/app/storage_explorer/resources/tips_and_resources.tsx

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  EuiAccordion,
+  EuiPanel,
+  EuiFlexItem,
+  EuiTitle,
+  EuiButton,
+  EuiCard,
+  EuiIcon,
+  EuiListGroup,
+  EuiFlexGroup,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { useApmRouter } from '../../../../hooks/use_apm_router';
+import { useApmParams } from '../../../../hooks/use_apm_params';
+import { useApmPluginContext } from '../../../../context/apm_plugin/use_apm_plugin_context';
+import {
+  getIndexManagementHref,
+  getStorageExplorerFeedbackHref,
+} from '../get_storage_explorer_links';
+
+export function TipsAndResources() {
```

Comment:
@bmorelli25 do you have any feedback for the content or links?

<img width="1494" alt="image" src="https://user-images.githubusercontent.com/5831975/196547751-90c24b6c-7aae-434c-81d5-13319440d285.png">

Reply:
Very cool! What about these for the three tabs?

**Reduce transactions**
Configure a more aggressive transaction sampling policy.
Transaction sampling lowers the amount of data ingested without negatively impacting the usefulness of that data.
[Learn more](https://www.elastic.co/guide/en/apm/guide/master/sampling.html)

**Reduce spans**
Enable span compression. Span compression saves on data and transfer costs by compressing multiple similar spans into a single span.
[Learn more](https://www.elastic.co/guide/en/apm/guide/master/span-compression.html)

**Manage the index lifecycle**
Customize your index lifecycle policies. Index lifecycle policies allow you to manage indices according to your performance, resiliency, and retention requirements.
[Learn more](https://www.elastic.co/guide/en/apm/guide/current/ilm-how-to.html)

Reply:
Thanks for the input! I've [updated](https://github.com/elastic/kibana/pull/143179/commits/b50e5a97522c86703809085dc8923f34154e9145) the cards to match the above.

Reply:
Thanks! Looks good.